### PR TITLE
Add CLI command for proposals

### PR DESCRIPTION
**Pull request by donce**
screen shot 2018-01-31 at 11 55 21

### CODE REVIEW

**Comment by donce** [comment]
Added provider id autocompletion (@tadovas)

**Comment by tadovas** [comment]
100 bravo tokens! LGTM

File: cmd/mysterium_client/cli/command.go [hunk 1]
```diff
@@ -220,6 +224,32 @@ func (c *Command) status() {
...
 }
 
+func (c *Command) proposals() {
+	proposals := c.fetchedProposals
+	info(fmt.Sprintf("Found %v proposals", len(proposals)))
+
+	for _, proposal := range proposals {
+		country := proposal.ServiceDefinition.LocationOriginate.Country
+		var countryString string
+		if country != nil {
```
Comment (tadovas) [inline cmd/mysterium_client/cli/command.go]: What about ternariy operator?  country == nil ? "default" : country

Comment (tadovas) [reply cmd/mysterium_client/cli/command.go]: RTFM for myself. Ignore this

**Pull request merged**